### ISSUE TITLE
Question about inference stability

### ISSUE THREAD

**Issue by IvanBarabanau** [taesungp/contrastive-unpaired-translation#36]
Could you, please, comment on the inference stability. I mean, does the translation output for the same image significantly varies from epoch to epoch or it is pretty much the same after a sufficient amount on training epochs?

**Comment by taesungp** [comment]
After a sufficient amount on training epochs, it stabilizes. In particular, toward the end of the epochs with decaying learning rate (specified with `--n_epochs_decay`), the learning rate will be close to zero, so the output will change little.
